pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 88
```

Comment:
We should try to be consistent with the line length setting (I think 100 is a good number, not too long, not too short). I would also configure isort here, instead of in setup.cfg.

You can probably mostly copy the implementation from mozregression here:

https://github.com/mozilla/mozregression/blob/master/pyproject.toml